src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/wizard.ts

```diff
@@ -101,6 +99,10 @@ export const useWizardStore = defineStore("wizard", {
     setWizardData(wizardData: WizardData): void {
       this.wizardData = { ...this.wizardData, ...wizardData };
     },
+    setCurrentStep(stage: string): void {
```

Comment:
same comment as above to make this into a PortalStage type so we don't put in random strings.